src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@use "./blocks/variables.scss" as *;
+@use "./blocks/reset.scss";
+@use "./blocks/header.scss";
+@use "./blocks/logo.scss";
+@use "./blocks/navigation.scss";
+@use "./blocks/card.scss";
+@use "./blocks/stars.scss";
```

Comment:
no need to name blocks with `_` in the beginning, it's common for mixins and variables

Reply:
_ removed from name of blocks